app/application.js

```diff
@@ -29,8 +29,14 @@ class RunConditionTableApplication {
         this.loggedUsers = {
             tokenToUserData: {},
         };
+
+        const args = process.argv.slice(2);
+        if (args[0] === 'noid') {
```

Comment:
For production, we shall not use other parameters. The `application.js` needs to check by itself if a configuration for openid exists, and if yes to use it.